.github/workflows/maven.yml

```
@@ -0,0 +1,61 @@
+name: CI build
+
+on:
+  push:
+    paths-ignore:
+      - docs/**
+  pull_request:
+    paths-ignore:
+      - docs/**
+
+jobs:
+  maven-build:
+
+    runs-on: [ubuntu-latest]
+    strategy:
+      matrix:
+        java: [1.8]
+
+    steps:
+    - uses: actions/checkout@v2
+    
+    - name: Set up JDK ${{ matrix.java }}
+      uses: actions/setup-java@v1
+      with:
+        java-version: ${{ matrix.java }}
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+    
+    - name: Cache maven dependencies
+      uses: actions/cache@v2
+      with:
+        path: ~/.m2
+        key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+        restore-keys: |
+          ${{ runner.os }}-maven-
+    
+    - name: Build with Maven
+      run: ./mvnw clean install
```

Providing  modules' list for building would make knowing errors, if encountered, easy to trace. 
```suggestion
      run: ./mvnw clean install -pl pantheon-bundle,pantheon-karaf-feature,pantheon-karaf-dist
```

---

I'm fine with this change, as long as we understand that if we ever add new maven modules to the larger project they will be ignored by the builder.

---

If we ever add a new module this would have to be updated as well. I am ok with clean install. Only thought was if a new module is added, then its easily understandable from the workflow file.